ont_fast5_api/conversion_tools/single_to_multi_fast5.py

```diff
@@ -107,12 +111,14 @@ def main():
                         help="Number of reads per multi-read file")
     parser.add_argument('-t', '--threads', type=int, default=1, required=False,
                         help="Number of threads to use")
-    parser.add_argument('--recursive', action='store_true',
+    parser.add_argument('-r', '--recursive', action='store_true',
                         help="Search recursively through folders for for single_read fast5 files")
+    parser.add_argument('--revert', action='store_true',
```

Comment:
Could we separate this out into a second PR? We haven't really thought about adding picopore-style functionality into here.

Reply:
yeap, sorry, didn't want to mess up the stuff. I'll try to sort this out this week. 
btw, I've implemented `--revert` as picopore isn't maintained any more. Beside, picopore's revert doesn't really work as advertised. And doing --revert on the fly on old fast5 during single_to_multi_fast5.py makes lots of sense as you spare lots of time and IO operations. Worked like charm for me :)